sound/soc/sof/mediatek/mt8195/mt8195.c

```diff
@@ -236,6 +236,18 @@ static int adsp_shared_base_ioremap(struct platform_device *pdev, void *data)
 	return ret;
 }
 
+static int mt8195_run(struct snd_sof_dev *sdev)
+{
+	u32 adsp_bootup_addr;
+
+	adsp_bootup_addr = SRAM_PHYS_BASE_FROM_DSP_VIEW;
+	dev_info(sdev->dev, "[ADSP] HIFIxDSP boot from base : 0x%08X\n",
+		 adsp_bootup_addr);
```

Comment:
I think this will be too verbose. Each PM suspend/resume cycle you will get this message. Use dev_dbg or either completely remove it.

Reply:
ok. I will use dev_dbg. thanks